scripts/eval.py

```diff
@@ -0,0 +1,286 @@
+            "setting has no effect."
+        )
+
+    barrier()
+
+    device = torch.device("cuda")
+
+    # Fill some configuration options.
+    cfg.model.precision = cfg.precision
+    cfg.device_train_batch_size = cfg.global_train_batch_size // get_world_size()
+    assert cfg.device_train_batch_size is not None  # for mypy
+    cfg.device_train_grad_accum = cfg.device_train_batch_size // cfg.device_train_microbatch_size
+    if cfg.optimizer.no_decay_norm_and_bias is not None:
+        log.warning(
+            "You set the deprecated config option `no_decay_norm_and_bias`. For compatibility, this"
+            "setting will take precedence over all other weight decay configurations. Please change"
+            "your config to use `decay_norm_and_bias` and `decay_embeddings` instead."
+        )
+        cfg.optimizer.decay_norm_and_bias = not cfg.optimizer.no_decay_norm_and_bias
+        cfg.optimizer.decay_embeddings = not cfg.optimizer.no_decay_norm_and_bias
+        cfg.optimizer.no_decay_norm_and_bias = None  # So nobody uses this by accident.
+
+    # Display and save configuration.
+    if get_global_rank() == 0:
+        if cfg.data.paths is not None and len(cfg.data.paths) < 50:
+            log.info("Configuration:")
+            log.info(cfg)
+        if not cfg.dry_run and (cfg.load_path is None or Path(cfg.load_path).parent != Path(cfg.save_folder)):
+            # Save config.
+            save_path = Path(cfg.save_folder) / "config.yaml"
+            if save_path.is_file() and not cfg.save_overwrite:
+                raise OLMoConfigurationError(f"{save_path} already exists, use --save_overwrite to overwrite")
+            else:
+                log.info(f"Saving config to {save_path}")
+                save_path.parent.mkdir(exist_ok=True, parents=True)
+                cfg.save(save_path)
+            del save_path
+
+    barrier()
+
+    # Maybe start W&B run.
+    if cfg.wandb is not None and (get_global_rank() == 0 or not cfg.wandb.rank_zero_only):
+        wandb_dir = Path(cfg.save_folder) / "wandb"
+        wandb_dir.mkdir(parents=True, exist_ok=True)
+        wandb.init(
+            dir=wandb_dir,
+            project=cfg.wandb.project,
+            entity=cfg.wandb.entity,
+            group=cfg.wandb.group,
+            name=cfg.wandb.name,
+            tags=cfg.wandb.tags,
+            config=cfg.asdict(exclude=["wandb"]),
+        )
+
+    barrier()
+
+    # Set seed.
+    seed_all(cfg.seed)
+
+    # # Construct data loader.
+    # train_loader = build_train_dataloader(cfg)
+    train_loader = None
+
+    # Construct evaluators.
+    evaluators = build_evaluators(cfg, device)
+    barrier()
+
+    if cfg.load_path is None:
+        raise OLMoConfigurationError("To run eval you must provide a load_path")
+    if 'step' in cfg.load_path.split('/')[-1]:
+        load_paths = [cfg.load_path]
+    else:
+        # This globbing does not work with remote paths.
```

Comment:
How is that problem handled then?

Reply:
Not handled. I will assume the checkpoints are on WEKA.

Reply:
At least throw an exception then.